app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions. Consider using `raise NotVaccinatedError('Visitor is not vaccinated.')`.